src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx

```diff
@@ -79,12 +86,28 @@ export class CancerSummaryChart extends React.Component<CancerSummaryChartProps,
 
     @observable.ref private barPlotTooltipModel: ITooltipModel | null;
     @observable.ref private scatterPlotTooltipModel: any | null;
+
+    @observable private barToolTipCounter = 0;
+    @observable private isBarPlotTooltipHovered = false;
+    @observable private shouldUpdatePosition = false; // Prevents chasing tooltip
+
     private svg: SVGElement;
     @observable mousePosition = { x:0, y:0 };
 
     constructor(props:CancerSummaryChartProps){
         super(props);
         this.tickFormat = this.tickFormat.bind(this);
+        this.tooltipMouseEnter = this.tooltipMouseEnter.bind(this);
```

Comment:
these calls to bind can now be handled more elegantly with use of @auto-bind decorator.   the ones you see now are legacy, from before we adopted the decorator.